File: src/js/exceptions.js
function DoStuff(inError) {
    if (inError)
        throw new Error("An error occured!");

    console.log("Doing stuff!");
}

try {
    DoStuff(false);
    DoStuff(true);
    DoStuff(false); // not reached
} catch (error) {
    console.log("ERROR: " + error.message);
}

File: src/kt/exceptions.kt
fun doStuff(inError: Boolean) {
    if (inError)
        throw Exception("An error occured!")

    println("Doing stuff!")
}

fun main(args: Array<String>) {
    try {
        doStuff(false)
        doStuff(true)
        doStuff(false) // not reached
    } catch (e: Exception) {
        print("ERROR: ${e.message}")
    }
}